### ISSUE TITLE
'statik --watch' does not monitor project definition modifications

### ISSUE THREAD

**Issue by tombreit** [thanethomson/statik#44]
When running Statik in `--watch` mode, it does not monitor modifications of the project configuration `config.yml` (e.g.  does not restart HTTP server).

**Comment by thanethomson** [comment]
This has been by design from the beginning so far, unfortunately. How often do you make changes to your project configuration?

**Comment by tombreit** [comment]
No, that's not a problem, I just did not know that this is the expected behavior while working on a `navigation_bar` static context var in my `config.yml`.  Perhaps a short notice on https://github.com/thanethomson/statik/wiki/Project-configuration would have prevented this notice;-)

**Comment by thanethomson** [comment]
Yes, that is something that I'd actually like to facilitate in future. It's a tricky feature, so I'll have to put it on the todo list for some point soon when I've got some spare time 😀

**Comment by pztrick** [comment]
Might also be worth monitoring the `statik` library module to restart the watch process also for us when we are developing. ;)

**Comment by jayvdb** [comment]
This probably requires some enhancements to https://github.com/thanethomson/httpwatcher , or using a different http server which allows more granular control over what is watched.
